pkg/apis/machine/v1alpha1/types.go

```diff
@@ -211,6 +211,24 @@ const (
 	MachineStateSuccessful MachineState = "Successful"
 )
 
+// MachineLastOperationType is a label for the last operation performed on a machine object.
+type MachineLastOperationType string
+
+// These are the valid statuses of machines.
+const (
+	// MachineLastOperationCreate indicates that the last operation was a create
+	MachineLastOperationCreate MachineLastOperationType = "Create"
+
+	// MachineLastOperationUpdate indicates that the last operation was an update
+	MachineLastOperationUpdate MachineLastOperationType = "Update"
+
+	// MachineLastOperationHealthCheck indicates that the last operation was a create
+	MachineLastOperationHealthCheck MachineLastOperationType = "HealthCheck"
```

Comment:
See comment above for health-check.

Reply:
Same answer as above. But if it doesn't make sense, we still could revert the changes.